service/init.go

```diff
@@ -399,7 +400,7 @@ func getPccRules(slice *protos.NetworkSlice, sessionRule *models.SessionRule) (p
 		rule.PccRuleId = strconv.FormatInt(int64(id), 10)
 		rule.Precedence = pccrule.Priority
 		if pccrule.Qos != nil {
-			qos.QosId = strconv.FormatInt(id, 10)
+			qos.QosId = "5GQos-" + strconv.FormatInt(id, 10)
```

Comment:
This is mapped to Qos Flow Id in SMF, so lets keep it back to integer value only